api/audio_options.h

```diff
@@ -80,6 +79,10 @@ struct RTC_EXPORT AudioOptions {
   absl::optional<bool> audio_network_adaptor;
   // Config string for audio network adaptor.
   absl::optional<std::string> audio_network_adaptor_config;
+  // Pre-initialize the ADM for recording when starting to send. Default to
+  // true.
+  // TODO(webrtc:13566): Remove this option. See issue for details.
+  absl::optional<bool> init_recording_on_send;
```

Comment:
I wonder if there will be side effects here, was it true before?

Reply:
It's used in this block:

    if (options_.init_recording_on_send.value_or(true) &&
        !engine()->adm()->RecordingIsInitialized() &&
        !engine()->adm()->Recording()) {
      if (engine()->adm()->InitRecording() != 0) {
        RTC_LOG(LS_WARNING) << "Failed to initialize recording";
      }
    }

Which means its absence previously was the same as if it's unset now.  The only thing that can change is to set it to false. But we we stuck this whole thing in an "#ifdef false" to avoid doing it, so we operate the same as if this were set to false.  So I suppose we could get rid of our fork and just set init_recording_on_send to false.